src/lib/utils/application-commands/ApplicationCommandRegistry.ts

```diff
@@ -66,6 +67,13 @@ export class ApplicationCommandRegistry {
 			}
 		}
 
+		if (options?.guildIds) {
```

Comment:
use `isNullishOrEmpty` from @sapphire/utilities so the whole if branch is skipped for empty arrays. 

Yes I know the for loop would be skipped with empty arrays, but lets not get into the for loop at all.